engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateMigrationService.java

```diff
@@ -431,11 +493,18 @@ public List<String> migrate() {
      * Existing templates are never updated by this migration service.
      */
     Template getOrCreateTemplate(List<String> warnings, String name, String extension, String description) {
+        String templateFromFS = loadResourceTemplate(name, extension);
         try {
+            boolean hasBeenUpdated = false;
             Template template = entityManager.createQuery("FROM Template WHERE name = :name", Template.class)
                     .setParameter("name", name)
                     .getSingleResult();
-            warnings.add(String.format("Template found in DB: %s", name));
+            if (!template.getData().equals(templateFromFS)) {
+                template.setData(templateFromFS);
+                template = entityManager.merge(template);
```

Comment:
Could you please check if that line can be removed?
```suggestion
```
This method is run within the boundaries of a JTA transaction so any change to `template` should be reflected in the DB (because of Hibernate's dirty check on managed entities) at the end of the transaction scope.

Reply:
Right I tested it, it works as you said. I remove this line.